### ISSUE TITLE
`aws_sdk_eventbridge` does not re-export `HttpResponse`

### ISSUE THREAD

**Issue by SAdams601** [smithy-lang/smithy-rs#3591]
`aws_sdk_eventbridge::operation::put_events::builders::PutEventsFluentBuilder::send` returns a `Result<PutEventsOutput, SdkError<PutEventsError, HttpResponse>>` but `HttpResponse` is not available without adding the `aws-smithy-runtime-api` crate as a dependency.

**Comment by SAdams601** [comment]
I returning the result of the `send` call from a function so I need to reference the type in the return type of that function.

**Comment by jdisanti** [comment]
Yes. It's defaulted in the `SdkError` type that is re-exported in aws-sdk-eventbridge, so you don't actually need to reference it.

**Comment by aajtodd** [comment]
`SdkError` and it's [constructor functions](https://github.com/smithy-lang/smithy-rs/blob/release-2024-06-19/rust-runtime/aws-smithy-runtime-api/src/client/result.rs#L341-L368) are public. You should be able to use them to create an instance.

**Comment by klawson88** [comment]
Can that approach be used to construct variants with the `raw` component (i.e the `R` type parameter), where that component is the same type used by the SDK (ex. `HttpResponse`)? It seems like that not is possible without importing `aws-smithy-runtime-api`, since those `raw` types are not re-exported by the service-level packages (in my case, it was `aws-sdk-s3`).

**Comment by JuiceDrinker** [comment]
Have the same use-case as @klawson88. Would like to create a wrapper around `aws-sdk-s3` which requires me to type my wrapper function explicilty as`Result<GetObjectOutput, SdkError<GetObjectError, HttpResponse>>` which is seemingly not possible without importing `aws-smithy-runtime-api`
